### DISCUSSION TITLE
Question about worldPos()

### DISCUSSION THREAD

**Discussion by martiansparlor** [replit/kaboom#859] [Q&A]
Hi, I'm still learning about the new parent/child system in the v3000, and I don't quite understand how to get an object position in a scene when the said object is created via the addLevel() function.
I tried the worldPos() comp I found in the documentation, but it still returns v2(0,0)
What would be the correct method ?

**Comment by martiansparlor** [comment]
Found the issue. For some reason, the position of an object is set at zero when called during add() or onAdd().
Since there is no longer a load() function in the component, the only fix I found is to get the pos during the first frame after the game launched.

**Reply by lajbel** [reply]
Hey! Sorry about no response. kaboom is no longer mantained, you can go to https://github.com/marklovers/kaplay
